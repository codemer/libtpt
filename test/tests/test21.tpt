@#
@# This is the @foreach test
@#
@set(list, "apple", "orange", "grape", "pear", "banana", "peach")\
@foreach (list) {
${.}
}
