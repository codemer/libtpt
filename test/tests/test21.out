apple
orange
grape
pear
banana
peach
